### PR TITLE
MARKET-11 SFTP Connector - Upload and Download Files

### PR DESCRIPTION
**Pull request by majeed-hm**

### CODE REVIEW

**Comment by ivy-rew** [comment]
Hi @andreasbalsiger 
I've checked the meta.json from a technical perspective: Can you please review the README.md plus the names+tags in the meta.json and give feedback to @majeed-hm ?
Thanks in advanced, 
Reguël

File: market/connector/sftp/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+**SFTP Connector** is an SFTP client which uses the SFTP protocol to transfer files securely to and from a remote computer.
+SFTP Connector is based on JSch [http://www.jcraft.com/jsch/] to handle the SFTP Connections and Operations.
```
Comment (andreasbalsiger) [inline market/connector/sftp/README.md:2]: May I suggest some other wording to give this intro text a marketing flair?

Suggestion:
Axon Ivy's SFTP Connector helps you to accelerate process automation initiatives by integrating secure file transfer into your process work. With this SFTP client you can easily transfer files securely to and from a remote computer. This connector: 
- uses the SFTP protocol
- is based on JSch [http://www.jcraft.com/jsch/] to handle the SFTP Connections and Operations
- is a pure Java implementation of SSH2
- allows you to connect to an sshd server and use port forwarding, X11 forwarding, file transfer, etc.
- makes it easy to integrate secure file transfer into your process work

**Review by andreasbalsiger — CHANGES_REQUESTED** [review]
Hi @majeed-hm (fyi: @ivy-rew) 
I have suggested some changes to give the intro a marketing flavour. Can you edit your readme.md accordingly. Many thanks, Andreas

File: market/connector/sftp/README.md (hunk 1 above)
Comment (majeed-hm) [reply market/connector/sftp/README.md]: Updated accordingly ;)

**Pull request merged**